src/Adapter/Product/Stock/Update/ProductStockUpdater.php

```diff
@@ -309,5 +310,16 @@ private function saveMovement(StockAvailable $stockAvailable, StockModification
                 'id_shop' => (int) $stockAvailable->id_shop,
             ]
         );
+
+        Hook::exec(
```

Comment:
Wow, that's quite a big issue if there was no hook previously

Reply:
There should be dependency injection here.